src/Uno.UITest.Helpers/AppInitializer.cs

```diff
@@ -268,7 +268,7 @@ private static IApp CreateiOSApp(bool alreadyRunningApp)
 
 			var app = alreadyRunningApp
 				? iOSConfig.ConnectToApp()
-				: iOSConfig.StartApp(Xamarin.UITest.Configuration.AppDataMode.DoNotClear);
```

Comment:
We are passing in mode here which defaults to Clear but we were previously passing DoNotClear specifically to the StartApp call.

Reply:
@kazo0 Sorry for the delay. I just took a look, and I'm not sure this one takes effect. From: https://docs.microsoft.com/en-us/dotnet/api/xamarin.uitest.configuration.iosappconfigurator.startapp?view=xamarin-uitest-sdk

> The app data mode. Whether to clear app data or not before app launch. **Currently only supported in Xamarin Test Cloud.**

This code path is reachable for local platform, not test cloud platform. I'll revert the changes to local platform.

Reply:
Hmm... Looks like the path I was actually testing is for local platform. Either Xamarin UI Test docs are outdated or there is something I don't understand here.